helm_deploy/apply-for-legalaid-api/README.md

```diff
@@ -0,0 +1,23 @@
+# Helm Deploy
+
+This describes the setup for helm and what is required to be in place to perform deploys.
+
+## CI configuration
+
+In order for a deploy to be successful, a set of configurations need to be in place:
+
+* `APPLICATON_DEPLOY_NAME` - name of the deployable app
+* `<ENV>_HOST` - base hostname for the deployable app
+* `GIT_CRYPT_KEY` - key to allow the decription of secrets required for deploy
+* `KUBE_ENV_<ENV>_CACERT`
+* `KUBE_ENV_<ENV>_NAME`
+* `KUBE_ENV_<ENV>_NAMESPACE`
+* `KUBE_ENV_<ENV>_CACERT`
```

Comment:
Is the environment ever going to be anything other than **UAT**? Assuming that UAT is being used to avoid existing similar environment variables, I think "UAT" should be used in this documentation rather than `<ENV>`:
```markdown
* `APPLICATON_DEPLOY_NAME` - name of the deployable app
* `UAT_HOST` - base hostname for the deployable app
* `GIT_CRYPT_KEY` - key to allow the decription of secrets required for deploy
* `KUBE_ENV_UAT_CACERT`
* `KUBE_ENV_UAT_NAME`
* `KUBE_ENV_UAT_NAMESPACE`
```

Reply:
<ENV> can be one of `uat`, `staging`, `production`.

Reply:
:+1: